.env-dist

```diff
@@ -1,5 +1,3 @@
-NODE_ENV=development
```

Comment:
We should avoid messing with this. `yarn run build:ssr` is *always* `NODE_ENV==='production'`. When using the React dev server (`yarn workspace client start`) it's *always* `NODE_ENV==='development'`. 
That's all you should need.